src/utils/featureConstants.ts

```diff
@@ -0,0 +1,165 @@
+import { GeoserverPostAttribute } from '../common/geoserver/models/featureType';
+
+export const srs: string = 'EPSG:4326';
+
+export const boundingBox = {
+  minx: -180,
+  maxx: 180,
+  miny: -90,
+  maxy: 90,
+  crs: srs,
+};
+
+export const attributesMapping = [
```

Comment:
check for required fields

Reply:
nullable are only: source_id, description. cities, countries